bigquery/tests/unit/test_client.py

```diff
@@ -2182,8 +2182,7 @@ def _row_data(row):
         self.assertEqual(req['data'], SENT)
 
     def test_create_rows_w_list_of_Rows(self):
-        from google.cloud.bigquery._helpers import Row
-        from google.cloud.bigquery.table import Table, SchemaField
+        from google.cloud.bigquery.table import Table, SchemaField, Row
```

Comment:
This should be 3 imports. From the [Google style guide][1]:

> Imports should be on separate lines.
> E.g.:
> ```
> Yes: import os
>      import sys
> No:  import os, sys
> ```

[1]: https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting

Reply:
Fixed these issues in 8cf4f53